libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/DataCharacteristicsHeader.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Stack, Text } from "@fluentui/react";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { dataCharacteristicsStyles } from "./DataCharacteristics.styles";
+
+export interface IDataCharacteristicsHeaderProps {
+  label: string;
+  labelListLength: number;
+  totalListLength: number;
+}
+
+export class DataCharacteristicsHeader extends React.Component<IDataCharacteristicsHeaderProps> {
+  public render(): React.ReactNode {
+    const classNames = dataCharacteristicsStyles();
+    return (
+      <Stack horizontal tokens={{ childrenGap: "l1" }} verticalAlign="center">
+        <Stack.Item>
+          <Text variant="xLarge">{this.props.label}</Text>
+        </Stack.Item>
+        <Stack.Item>
+          <Text variant="medium">
+            {this.props.labelListLength}/{this.props.totalListLength}{" "}
+            {localization.InterpretVision.Dashboard.examples}
+          </Text>
+        </Stack.Item>
+        <Stack.Item>
+          <Stack horizontal className={classNames.progressBar}>
```

Comment:
It's not an actual progress bar that dynamically updates, it's just a static view of what percentage of the total examples each label comprises. I just call it a progress bar in my styles because it looks like one